synapse/cores/ram.py

```diff
@@ -175,6 +191,58 @@ def _getSizeByProp(self, prop, valu=None, mintime=None, maxtime=None):
 
         return len(rows)
 
+    def _getCoreType(self):
+        return 'ram'
+
+    def _getBlobValu(self, key):
+        ret = self._blob_store.get(key)
+        return ret
+
+    def _setBlobValu(self, key, valu):
+        self._blob_store[key] = valu
+        self._blob_savebus.fire('syn:core:{}:blob:set'.format(self._getCoreType()), key=key, valu=valu)
+
+    def _hasBlobValu(self, key):
+        return key in self._blob_store
+
+    def _delBlobValu(self, key):
+        ret = self._blob_store.pop(key)
+        self._blob_savebus.fire('syn:core:{}:blob:del'.format(self._getCoreType()), key=key)
+        return ret
+
+    def _onSetBlobValu(self, mesg):
+        key = mesg[1].get('key')
+        valu = mesg[1].get('valu')
+        self._setBlobValu(key, valu)
+
+    def _onDelBlobValu(self, mesg):
+        key = mesg[1].get('key')
+        self._delBlobValu(key)
+
+    def _setSaveFd(self, fd, load=True, fini=False):
+        '''
```

Comment:
If the blob events were both "not storage layer specific" and passed over the existing load/save buses, we could likely do away with the ram specific certain of this function.